chia/server/rate_limit_numbers.py

```diff
@@ -0,0 +1,178 @@
+            ProtocolMessageTypes.respond_plots: RLSettings(10, 100 * 1024 * 1024),
+            ProtocolMessageTypes.plot_sync_start: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.plot_sync_loaded: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.plot_sync_removed: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.plot_sync_invalid: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.plot_sync_keys_missing: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.plot_sync_duplicates: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.plot_sync_done: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.plot_sync_response: RLSettings(3000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.coin_state_update: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.register_interest_in_puzzle_hash: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.respond_to_ph_update: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.register_interest_in_coin: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.respond_to_coin_update: RLSettings(1000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.request_ses_hashes: RLSettings(2000, 1 * 1024 * 1024),
+            ProtocolMessageTypes.respond_ses_hashes: RLSettings(2000, 1 * 1024 * 1024),
+            ProtocolMessageTypes.request_children: RLSettings(2000, 1024 * 1024),
+            ProtocolMessageTypes.respond_children: RLSettings(2000, 1 * 1024 * 1024),
+        },
+    },
+    2: {
+        "default_settings": RLSettings(100, 1024 * 1024, 100 * 1024 * 1024),
+        "non_tx_freq": 1000,  # There is also a freq limit for many requests
+        "non_tx_max_total_size": 100 * 1024 * 1024,  # There is also a size limit for many requests
+        "rate_limits_tx": {
+            ProtocolMessageTypes.request_block_header: RLSettings(500, 100),
+            ProtocolMessageTypes.respond_block_header: RLSettings(500, 500 * 1024),
+            ProtocolMessageTypes.reject_header_request: RLSettings(500, 100),
+            ProtocolMessageTypes.request_removals: RLSettings(5000, 50 * 1024, 10 * 1024 * 1024),
+            ProtocolMessageTypes.respond_removals: RLSettings(5000, 1024 * 1024, 10 * 1024 * 1024),
+            ProtocolMessageTypes.reject_removals_request: RLSettings(500, 100),
+            ProtocolMessageTypes.request_additions: RLSettings(50000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.respond_additions: RLSettings(50000, 100 * 1024 * 1024),
+            ProtocolMessageTypes.reject_additions_request: RLSettings(500, 100),
+            ProtocolMessageTypes.reject_header_blocks: RLSettings(1000, 100),
+            ProtocolMessageTypes.respond_header_blocks: RLSettings(5000, 2 * 1024 * 1024),
+            ProtocolMessageTypes.request_block_headers: RLSettings(5000, 100),
```

Comment:
if someone claims to support this (or the other two) new message, but without claiming support for rate limits v2, I suppose there would be a problem, right? (since the rate limit for this message is only specified in v2)

Reply:
yeah.. i guess that would be an incompatible combination. Rate limits V2 is a prerequisite for block headers